src/components/PrizePage/Prizes.js

```diff
@@ -132,7 +132,8 @@ export default function Prizes() {
 	}
 	return (
 		<>
-			<Container maxWidth='md'>
+			<p style={{ fontSize: '24px', paddingBottom: '50px', paddingLeft: '100px' }}>Coming soon...</p>
```

Comment:
It would be better to make a **Coming Soon** component and then use that component in the different files. If they have different styles, you can pass in the extra styles as props.